include/eve/module/core/regular/remainder.hpp

```diff
@@ -7,11 +7,103 @@
 //==================================================================================================
 #pragma once
 
-#include <eve/detail/overload.hpp>
+#include <eve/arch.hpp>
+#include <eve/traits/overload.hpp>
+#include <eve/module/core/decorator/core.hpp>
+#include <eve/module/core/regular/div.hpp>
+#include <eve/module/core/regular/is_not_infinite.hpp>
+#include <eve/module/core/regular/is_nez.hpp>
+#include <eve/module/core/regular/is_eqz.hpp>
+#include <eve/module/core/regular/is_infinite.hpp>
+#include <eve/module/core/regular/is_unordered.hpp>
+#include <eve/module/core/regular/fanm.hpp>
+#include <eve/module/core/regular/nearest.hpp>
+#include <eve/module/core/constant/allbits.hpp>
 
 namespace eve
 {
-EVE_MAKE_CALLABLE(remainder_, remainder);
-}
+  template<typename Options>
+  struct remainder_t : elementwise_callable<remainder_t, Options>
+  {
+    template<value T,  value U>
+    constexpr EVE_FORCEINLINE common_value_t<T, U> operator()(T a, U b) const
+    { return EVE_DISPATCH_CALL(a, b); }
+
+    EVE_CALLABLE_OBJECT(remainder_t, remainder_);
+  };
 
-#include <eve/module/core/regular/impl/remainder.hpp>
+//================================================================================================
+//! @addtogroup core_arithmetic
+//! @{
+//!   @var remainder
+//!   @brief Computes the  remainder after division for floating entries similarly to std::remainder
+//!   for floating entries.
+//!
+//!   **Defined in Header**
+//!
+//!   @code
+//!   #include <eve/module/core.hpp>
+//!   @endcode
+//!
+//!   @groupheader{Callable Signatures}
+//!
+//!   @code
+//!   namespace eve
+//!   {
+//!      template< eve::signed_value T,  eve::signed_value U>
```

Comment:
why signed_value while the implemenation says value ?

Reply:
remainder needs signed value as the computed remainder can be negative
corrected